_posts/2015-09-04-channel.html

```diff
@@ -3614,6 +3614,345 @@
 	}
 }</code></pre>
 
+		<h3>Video caption</h3>
+
+		<h4>List all captions</h4>
+		<p>List all captions to a given videoId</p>
+
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td>
+					<a>https://api.ustream.tv/videos/VIDEO_ID/captions.FORMAT</a>
+					(format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Parameters</h6>
+		<p>None.</p>
+
+		<h6>Response</h6>
+		<p>A list of key-value pairs under a "captions" key. Example:</p>
+		<pre><code class="json">{
+    "captions": [
+        {
+            "is_default": true,
+            "is_public": false,
+            "language": "en-00",
+            "status": "eu dolor exercitation est do",
+            "video_id": "10001"
+        },
+        {
+            "is_default": false,
+            "is_public": false,
+            "language": "en-UK",
+            "status": "dolor irure velit aliquip",
+            "video_id": "10001"
+        }
+    ]
+}</code></pre>
+
+		<h6>Error codes</h6>
+		<table class="table responsive">
+			<tr>
+				<th>error value</th>
+				<th>HTTP response code</th>
+				<th width="100%">Error conditions</th>
+			</tr>
+			<tr>
+				<td>unauthorized</td>
+				<td>401 Authorization required</td>
+				<td>The video is protected, only the owner with a valid token can access it.</td>
+			</tr>
+			<tr>
+				<td>lack_of_ownership</td>
+				<td>403 Forbidden</td>
+				<td>The given video does not belong to the user.</td>
+			</tr>
+			<tr>
+				<td>not_found</td>
+				<td>404 Not Found</td>
+				<td>The given video does not exist.</td>
+			</tr>
+		</table>
+
+		<h4>Show caption details</h4>
+		<table class="table fill-bg-light1 table-chromeless">
+			<tr>
+				<td class="verb">GET</td>
+				<td>
+					<a>https://api.ustream.tv/videos/VIDEO_ID/captions/LANGUAGE.FORMAT</a>
+					(format can be json or xml)</td>
+			</tr>
+		</table>
+
+		<h6>Response</h6>
+		<p>Key-value pairs. Example:</p>
+		<pre><code class="json">{
+    "is_default": true,
+    "is_public": false,
+    "language": "en-00",
+    "status": "active",
+    "video_id": "10001"
+}</code></pre>
+
+		<p>Explanation of some values:</p>
+		<table class="table responsive">
+			<tr>
+				<th>key</th>
+				<th>type</th>
+				<th width="100%">Description</th>
+			</tr>
+			<tr>
+				<td>status</td>
+				<td>string</td>
+				<td>Possible values are: processing and active</td>
```

Comment:
Vagy "Possible values: processing and active", vagy "Possible values are processing and active" legyen. Az előbbi jobban tetszik számomra (és a többi helyen is így szerepel a doksiban).